components/net/http_loader.rs

```diff
@@ -644,7 +644,12 @@ pub fn http_fetch(
         };
     }
 
-    // TODO redirect_end: last byte of response of last redirect
+    // redirect_end: last byte of response of last redirect
+    context
+        .timing
+        .lock()
+        .unwrap()
+        .set_attribute(ResourceAttribute::RedirectEnd);
```

Comment:
Unfortunately I think the TODO was misleading. Much like my review comment in #22319, step 20 of https://w3c.github.io/resource-timing/#processing-model suggests that this should be set in a different part of the code, and will be very similar to how we implement responseEnd (#21263). This part of the code has only just received the HTTP headers, not the response body.